restli-tools/src/main/java/com/linkedin/restli/tools/compatibility/CompatibilityInfoMap.java

```diff
@@ -166,6 +166,21 @@ public String createSummary()
     return summaryMessage.toString();
   }
 
+  public String createPegasusSchemaSummary()
```

Comment:
Not used anywhere?

Reply:
It is supposed to use to create summary report. When I edited code, we accidentally removed the usage. 
Updated.

Reply:
No more usage, deleted.